components/bl-signature-pad-component/src/leap/signature_pad.umd.min.js

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
We have a little misunderstanding I think :)

I was talking about `lib` directory(library)